audformat/core/database.py

```diff
@@ -535,6 +572,16 @@ def join_field(
             assert_equal(other, 'license')
             assert_equal(other, 'usage')
 
+        # can only join databases with relatvie paths
+        for database in [self] + others:
+            if not database.is_portable():
+                raise RuntimeError(
+                    f"You can only use update with databases "
```

Comment:
I would either change to "You can only update with databases...." or "You can only use update() with databases..."

Reply:
I used "You can only update with... " already at another place, so I changed it to that one.